### PR TITLE
make sure k8s schema supports everything in marathon schema

### PR DESCRIPTION
**Pull request by chlgit**
Add the configurations shown in `marathon_schema.json`.

### CODE REVIEW

File: paasta_tools/cli/schemas/kubernetes_schema.json [hunk 1]
```diff
@@ -334,6 +366,33 @@
...
+                },
+                "dependencies_reference": {
+                    "type": "string"
+                },
+                "extra_docker_args": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    }
+                },
```
Comment (vkhromov) [inline paasta_tools/cli/schemas/kubernetes_schema.json]: k8s doesn't support extra docker args

File: paasta_tools/cli/schemas/kubernetes_schema.json [hunk 2]
```diff
@@ -254,6 +273,13 @@
                     },
                     "additionalProperties": true
                 },
+                "marathon_shard": {
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "previous_marathon_shards": {
+                    "type": "array"
+                },
```
Comment (vkhromov) [inline paasta_tools/cli/schemas/kubernetes_schema.json]: `k8s_shard`?

File: paasta_tools/cli/schemas/kubernetes_schema.json (hunk 1 above)
Comment (chlgit) [reply paasta_tools/cli/schemas/kubernetes_schema.json]: Yeah, the point for this change is to allow user (or us) using `git mv` for marathon-{cluster}.yaml to kubernetes-{cluster}.yaml and clean up afterward.

File: paasta_tools/cli/schemas/kubernetes_schema.json (hunk 2 above)
Comment (chlgit) [reply paasta_tools/cli/schemas/kubernetes_schema.json]: Right, we also need to modify this later.

**Comment by solarkennedy** [comment]
👍 I is ok that we don't support these features yet. As we move stagef they will "magically" start working or just not apply, and we can clean up on a second pass. This change is just to make it easier rename files back and forth.

**Pull request merged**